Add improvements to API docs. (#2578)
wso2/docs-is: en/docs/develop/apis/application-rest-api.md

??? Note "Click for instructions"
    Follow the steps given below to try out the REST APIs with your local instance of WSO2 Identity Server (WSO2 IS).

>     1.  Click on **Authorize** button and provide desired values for authentication.

Reviewer: ```suggestion
    1.  Click **Authorize** and provide the desired values for authentication.
```
